hw2pt2_concatenation.py

```diff
@@ -16,12 +14,12 @@
     links.append(link.get('href'))
 
 for link in links:
-    print('https://samples.adsbexchange.com/readsb-hist/2022/05/01/'+link)
+    if '.json.gz' in link:
+        print('https://samples.adsbexchange.com/readsb-hist/2022/05/01/'+link)
 
-print(len(links))
+datafile_path = '/Users/alexguay/Workspace/OperationRavenousSuckerPunch-2/data.txt'
 
-f.write('https://samples.adsbexchange.com/readsb-hist/2022/05/01/'+link)
+download_url(datafile_path)
```

Comment:
this doesn't make sense.

you need this to run on each link. so basically _instead_ of printing you want to do _`download_url`_.